packages/base/src/types/AnimationModes.js

```diff
@@ -0,0 +1,7 @@
+const AnimationModes = {
```

Comment:
Should be AnimationMode (not modes), similar to ButtonType.

Reply:
yes, done